3.Promises-draw-circle/draw-circle-task-3/script.js

```diff
@@ -0,0 +1,20 @@
+function showCircle(cx, cy, radius) {
+  let div = document.createElement("div");
+  div.style.width = 0;
+  div.style.height = 0;
+  div.style.left = cx + "px";
+  div.style.top = cy + "px";
+  div.className = "circle";
+  document.body.append(div);
+
+  return new Promise((resolve) => {
+    setTimeout(() => {
+      div.style.width = radius * 2 + "px";
+      div.style.height = radius * 2 + "px";
```

Comment:
![image](https://user-images.githubusercontent.com/11594423/121925975-bf3bd280-cd3d-11eb-8afe-28edf1692275.png)
![image](https://user-images.githubusercontent.com/11594423/121926010-c662e080-cd3d-11eb-8a0a-fed6bde12e48.png)
do you see a pattern? I would move to separate function

Reply:
yes, understood!

btw do you think it would be ok to write it like this:
```
div.style.width = div.style.height = 0
div.style.width = div.style.height = radius * 2 + "px";
```

Reply:
no. Write as simple and explicit as possible. The less effort you make during the code reading, the better. hint: 
```
function makeSquare(element, sizePx)

Reply:
oh ok, I overthink a little. Thank you!